### PR TITLE
Introduce PowerNV platform

### PR DESCRIPTION
**Pull request by k0da**
We treated PowerNV as CHRP platform for a while, for compatibility
reasons.

This introduces board_powernv

Signed-off-by: Dinar Valeev <dvaleev@suse.com>

### CODE REVIEW

File: library/general/src/modules/Arch.rb [hunk 1]
```diff
@@ -298,6 +293,11 @@ def board_chrp
       ppc && board_compatible == "CHRP"
     end
 
+    # true for all "CHRP" ppc boards
```
Comment (jreidinger) [inline library/general/src/modules/Arch.rb]: so all chrp boards are powerNV? what is different to boarch_chrp? or just copy and paste?
Can you please also add to comment link to some documentation of this kind of ppc board?

File: package/yast2.changes [hunk 2]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Mar 10 10:15:13 UTC 2016 - dvaleev@suse.com
+
+- Introduce OPAL platform (boo#970582)
```
Comment (jreidinger) [inline package/yast2.changes]: and here is name opal platform. It is a bit confusing for me.

**Comment by jreidinger** [comment]
pull request is not mergable. For me the most important note is about documentation for this board, so we know where to look if there is some problems with it.

File: package/yast2.changes (hunk 2 above)
Comment (kobliha) [inline package/yast2.changes]: +1 PLS be consistent, all the time

**Comment by kobliha** [comment]
The bug in bugzilla says what, but doesn't say why or does not link any source of information. Moreover, FATE should be more appropriate as we need QA testing and documentation here.

File: library/general/src/modules/Arch.rb (hunk 1 above)
Comment (k0da) [inline library/general/src/modules/Arch.rb]: Just copy paste.. Fixed

File: package/yast2.changes (hunk 2 above)
Comment (k0da) [inline package/yast2.changes]: Fixed

**Comment by k0da** [comment]
What kind of information is required? 

We need that platform so we could partition baremetal systems differently. Basically those are kexec systems no stage1 is required

**Comment by jreidinger** [comment]
@k0da I really would like to see in arch file link to some kind of platform/architecture specification by its vendor. So we can check it in future, if there is problem with such platform. Also to check if it is still supported platform by its vendor.

**Comment by k0da** [comment]
updated

**Comment by jreidinger** [comment]
LGTM, so just please merge master into branch so it is mergable and I will merge this one

**Comment by k0da** [comment]
updated

**Comment by jreidinger** [comment]
thanks. merged

**Pull request merged**